webapp/urls.py

```diff
@@ -1,10 +1,10 @@
 # Modules
 from django.conf.urls import url
-from django_json_redirects import load_redirects
+from django_yaml_redirects import load_redirects
 from django_template_finder_view import TemplateFinder
 
 # Local
-from .views import custom_404, custom_500
+from webapp.views import custom_404, custom_500
```

Comment:
Same point about relative namespacing that I made above - I think, conceptually, the `urls.py` file should be saying "from the views file in this module" rather than "from the views file in the webapp module".